R/slack.R

```diff
@@ -1,62 +1,52 @@
 ## Send slack messages!
 
-slack_post_success <- function(dat, config) {
+post_success <- function(dat, config) {
```

Comment:
I think this a good way of handling this for now - posts to both rather than either, and we minimal extra debt.  When I do this internal structure refactor we'll have defined "phases" of execution and that will include something "hookable" - we can then move this stuff out there somehow perhaps.  But very open to other ideas of how to sort that out